File: POH_V7/b1.rb
q = Array.new
n = gets.to_i
n.times{
  q << gets.chomp.split(" ")
}

p = Array.new
m = gets.to_i
m.times{
  p << gets.chomp.split(" ")
}

s = q.size - p.size + 1
t = false
s.times {|i|
  s.times {|j|
    r = Array.new
    (p.size).times {|k|
      r << q[i+k].slice(j,p.size)
    }
    if r == p
      puts "#{i} #{j}"
      t = true
      break
    end
  }
  break if t == true
}

File: POH_V7/Python/b1.py
n = int(input())
q = [ input().split() for i in range(n) ]
m = int(input())
p = [ input().split() for i in range(m) ]
s = len(q) - len(p) + 1
t = False
for i in range(s):
    for j in range(s):
        r = [ q[i+k][j:len(p)+j] for k in range(len(p)) ]
        if r == p:
            print(str(i),str(j))
            t = True
            break
    if t == True:
        break
